content/en/docs/v3.5/quickstart.md

```diff
@@ -1,61 +1,122 @@
 ---
 title: Quickstart
 weight: 900
-description: Get etcd up and running in less than 5 minutes!
+description: Run etcd Locally Under 5 minutes!
 ---
 
-Follow these instructions to locally install, run, and test a single-member
-cluster of etcd:
+This guide walks through installing and running a single-node etcd cluster locally using pre-built binaries - no containers or cloud set up required.
 
- 1. Install etcd from pre-built binaries or from source. For details, see
-    [Install][].
+## Prerequisites
 
-    {{% alert color="warning" %}}**Important**: Ensure that you perform the last
-    step of the installation instructions to verify that `etcd` is in your path.
-    {{% /alert %}}
+Make sure you're on a supported platform before getting started:
 
- 2. Launch `etcd`:
+- Operating System: Ubuntu 22.04+ or macOS 11+. Windows users can use WSL2.
```

Comment:
Where did you get these operating systems? This is what we have as supported platform, https://etcd.io/docs/v3.5/op-guide/supported-platform/ I would suggest to provide a link here. Keeping one page for for supported platforms will avoid need for dual maintenance as well.

Reply:
@spzala I agree that normally maintaining duplicate information isn't worth it. However, the Quick Start guide is a special case. The prereqs here should be only the OSes that can be installed using the quick start procedure. The regular installation guide should be a detailed guide to all installations.

Reply:
@dwelsch-esi thanks for your thoughts here and other places! I agree and I also like as much as information on the same page (vs links). It's just that if we run into two set of instructions (due to dual maintenance when we forgot to make changes at both places when something change) that doesn't match that will be bad.